cmd/github-labels/labels.yaml.in

```diff
@@ -361,6 +361,11 @@ labels:
     category: vendor
     color: DEFAULT_COLOUR
 
+  - name: needs-triage
```

Comment:
Personally, I prefer `needs-review`. Ack that 'triage' is maybe less obvious

Reply:
Yeah - I was hoping someone would agree :smile:. Branch updated (with a Fixes too! :)

Reply:
review sounds more like a PR item, not an issue to me.  I prefer triage, but don't care *that* much :)

Reply:
Thanks @egernst. One vote for and one against so do we need a tie-breaker?

| Label name | Pros | Cons | Applies to issues | Applies to PRs |
|-|-|-|-|-|
| `triage` | Implies "initial" [1]. Well understood for native English speakers. | Could be confusing if non-native English speaker. | Yes | "yes", but would sound "wrong" |
| `review` | More easily understood by all | ? | yes | yes |
| `assessment` | Implies "initial" [1] | ? |  yes | yes |

Note: If  we really want `triage`, we _might_ be able to discount the non-native English speaker argument given that all the labels are documented?

Any thoughts @kata-containers/architecture-committee?

---

[1] - Which works well for issues.

Reply:
Can you attach emojis to labels? 🚑 :-)
I'm fine with `triage` if we don't get a deciding vote.

Reply:
Better than that - you can use emoji's *as labels!* :smile:

Reply:
But that could get confusing what with the similarity of things like :chestnut: with "other things" :grin:

Reply:
@egernst - soz - this got merged before we really finished the chat. I've applied `needs-review` to all the repos but can change to `needs-triage` if you want (along with a new PR to update the master YAML this PR changed)?